src/sentry/static/sentry/app/index.js

```diff
@@ -19,11 +20,11 @@ import moment from 'moment';
 
 import 'app/utils/emotion-setup';
 
-import Main from 'app/main';
+import * as il8n from 'app/locale'; // XXX(webpack4): Currently all locales get bundled, would be good to have this async?
```

Comment:
Will bundling all the translations appreciably increase the main bundle?

Reply:
They're currently already being bundled